🐞 - Missed import `TuiIcon` during migration of `tuiTextfieldCustomContent`

### Reproduction url

https://github.com/taiga-family/maskito/blob/main/projects/demo/src/pages/recipes/placeholder/examples/3-date/component.ts

### Description

```ts
import {ChangeDetectionStrategy, Component} from '@angular/core';
import {FormsModule} from '@angular/forms';
import {MaskitoDirective} from '@maskito/angular';
import {TuiInputModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';

import mask from './mask';

@Component({
    selector: 'placeholder-doc-example-3',
    imports: [
        FormsModule,
        MaskitoDirective,
        TuiInputModule,
        TuiTextfieldControllerModule,
    ],
    template: `
        <tui-input
            tuiTextfieldCustomContent="@tui.calendar"
            [style.max-width.rem]="20"
            [(ngModel)]="value"
        >
            Enter date
            <input
                inputmode="numeric"
                tuiTextfieldLegacy
                [maskito]="maskitoOptions"
            />
        </tui-input>
    `,
    changeDetection: ChangeDetectionStrategy.OnPush,
})
export class PlaceholderDocExample3 {
    protected readonly maskitoOptions = mask;
    protected value = '';
}
```

are migrated to 
```ts
import { TuiInput } from "@taiga-ui/core";
import {ChangeDetectionStrategy, Component} from '@angular/core';
import {FormsModule} from '@angular/forms';
import {MaskitoDirective} from '@maskito/angular';
import mask from './mask';

@Component({
    selector: 'placeholder-doc-example-3',
    imports: [
        FormsModule,
        MaskitoDirective,
        TuiInput
    ],
    template: `
        <!-- TODO: (Taiga UI migration) tui-input migration (see https://taiga-ui.dev/components/input):
     - "[style" is an unrecognized attribute and was placed on <tui-textfield>. Move it to <input tuiInput> if it targets the native element.
-->
        <tui-textfield [style.max-width.rem]="20">
        <input
                inputmode="numeric"
                tuiInput
                [maskito]="maskitoOptions" [(ngModel)]="value"/>

        <tui-icon icon="@tui.calendar" />
        </tui-textfield>
    `,
    changeDetection: ChangeDetectionStrategy.OnPush,
})
export class PlaceholderDocExample3 {
    protected readonly maskitoOptions = mask;
    protected value = '';
}
```

It throws
```
✘ [ERROR] NG8001: 'tui-icon' is not a known element:
1. If 'tui-icon' is an Angular component, then verify that it is included in the '@Component.imports' of this component.
2. If 'tui-icon' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@Component.schemas' of this component to suppress this message. [plugin angular-compiler]

    projects/demo/src/pages/recipes/placeholder/examples/3-date/component.ts:24:8:
      24 │         <tui-icon icon="@tui.calendar" />
         ╵         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

We should also add `TuiIcon` import!
Don't use `tui-textfield[iconEnd]` because it can be already occupied by `[tuiChevron]`!

### Taiga UI version

5.3.0

### Browser Used

Chrome

### OS Used

macOS

--- a/projects/cdk/schematics/ng-update/v5/index.ts
+++ b/projects/cdk/schematics/ng-update/v5/index.ts
@@ -12,6 +12,7 @@ import {saveActiveProject} from 'ng-morph';
 import {TAIGA_VERSION} from '../../ng-add/constants/versions';
 import {type TuiSchema} from '../../ng-add/schema';
 import {FINISH_SYMBOL, START_SYMBOL, titleLog} from '../../utils/colored-log';
+import {saveAddedImports} from '../../utils/add-import-to-closest-module';
 import {
     formatMigrationStats,
     type MigrationStepTiming,
@@ -132,6 +133,7 @@ function main(options: TuiSchema, timings: MigrationStepTiming[]): Rule {
             timings,
         );
 
+        saveAddedImports(options);
         fileSystem.commitEdits();
         saveActiveProject();
 

--- a/projects/cdk/schematics/ng-update/v5/index.ts
+++ b/projects/cdk/schematics/ng-update/v5/index.ts
@@ -11,8 +11,8 @@ import {saveActiveProject} from 'ng-morph';
 
 import {TAIGA_VERSION} from '../../ng-add/constants/versions';
 import {type TuiSchema} from '../../ng-add/schema';
-import {FINISH_SYMBOL, START_SYMBOL, titleLog} from '../../utils/colored-log';
 import {saveAddedImports} from '../../utils/add-import-to-closest-module';
+import {FINISH_SYMBOL, START_SYMBOL, titleLog} from '../../utils/colored-log';
 import {
     formatMigrationStats,
     type MigrationStepTiming,

--- a/projects/cdk/schematics/ng-update/v5/steps/migrate-templates.ts
+++ b/projects/cdk/schematics/ng-update/v5/steps/migrate-templates.ts
@@ -2,7 +2,6 @@ import {type DevkitFileSystem, type UpdateRecorder} from 'ng-morph';
 
 import {ALL_TS_FILES} from '../../../constants';
 import {type TuiSchema} from '../../../ng-add/schema';
-import {saveAddedImports} from '../../../utils/add-import-to-closest-module';
 import {
     infoLog,
     REPLACE_SYMBOL,
@@ -148,8 +147,6 @@ export function migrateTemplates(fileSystem: DevkitFileSystem, options: TuiSchem
     });
 
     fileSystem.commitEdits();
-
-    saveAddedImports(options);
 
     !options['skip-logs'] &&
         successLog(`${SMALL_TAB_SYMBOL}${SUCCESS_SYMBOL} templates migrated \n`);

--- a/projects/cdk/schematics/ng-update/v5/tests/schematic-migrate-legacy-custom-content.spec.ts
+++ b/projects/cdk/schematics/ng-update/v5/tests/schematic-migrate-legacy-custom-content.spec.ts
@@ -127,6 +127,103 @@ describe('ng-update legacy tuiTextfieldCustomContent across controls', () => {
                     '<tui-select formControlName="value" tuiTextfieldCustomContent="@tui.calendar">Label</tui-select>',
             }),
         );
+
+        it(
+            'works with inline template (static icon => TuiInput + TuiIcon)',
+            migrate({
+                component: `
+                    import {ChangeDetectionStrategy, Component} from '@angular/core';
+                    import {FormsModule} from '@angular/forms';
+                    import {MaskitoDirective} from '@maskito/angular';
+                    import {TuiInputModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';
+
+                    import mask from './mask';
+
+                    @Component({
+                        selector: 'placeholder-doc-example-3',
+                        imports: [
+                            FormsModule,
+                            MaskitoDirective,
+                            TuiInputModule,
+                            TuiTextfieldControllerModule,
+                        ],
+                        template: \`
+                            <tui-input
+                                tuiTextfieldCustomContent="@tui.calendar"
+                                [style.max-width.rem]="20"
+                                [(ngModel)]="value"
+                            >
+                                Enter date
+                                <input
+                                    inputmode="numeric"
+                                    tuiTextfieldLegacy
+                                    [maskito]="maskitoOptions"
+                                />
+                            </tui-input>
+                        \`,
+                        changeDetection: ChangeDetectionStrategy.OnPush,
+                    })
+                    export class PlaceholderDocExample3 {
+                        protected readonly maskitoOptions = mask;
+                        protected value = '';
+                    }
+                `,
+            }),
+        );
+
+        it(
+            'works with inline template (dynamic content => TuiInput + TuiIcon + PolymorpheusOutlet)',
+            migrate({
+                component: `
+                    import {ChangeDetectionStrategy, Component} from '@angular/core';
+                    import {FormsModule} from '@angular/forms';
+                    import {MaskitoDirective} from '@maskito/angular';
+                    import {TuiFlagPipe} from '@taiga-ui/core';
+                    import {TuiInputModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';
+
+                    import mask from './mask';
+
+                    @Component({
+                        selector: 'placeholder-doc-example-2',
+                        imports: [
+                            FormsModule,
+                            MaskitoDirective,
+                            TuiFlagPipe,
+                            TuiInputModule,
+                            TuiTextfieldControllerModule,
+                        ],
+                        template: \`
+                            <tui-input
+                                [style.max-width.rem]="20"
+                                [tuiTextfieldCustomContent]="usFlag"
+                                [(ngModel)]="value"
+                            >
+                                Enter US phone number
+                                <input
+                                    inputmode="tel"
+                                    tuiTextfieldLegacy
+                                    [maskito]="maskitoOptions"
+                                />
+
+                                <ng-template #usFlag>
+                                    <img
+                                        alt="Flag of the United States"
+                                        width="28"
+                                        [src]="'US' | tuiFlag"
+                                        [style.border-radius.%]="50"
+                                    />
+                                </ng-template>
+                            </tui-input>
+                        \`,
+                        changeDetection: ChangeDetectionStrategy.OnPush,
+                    })
+                    export class PlaceholderDocExample2 {
+                        protected readonly maskitoOptions = mask;
+                        protected value = '';
+                    }
+                `,
+            }),
+        );
     });
 
     afterEach(() => resetActiveProject());

